plugin.video.vrt.nu/service.py

```diff
@@ -8,8 +8,7 @@
 
```

Comment:
What does the service do exactly?

Reply:
The service is not new, but it detects settings-changes and invalidates tokens and caches.

Since credentials could have changed, we should avoid using old tokens and e.g. a new favourites cache.